tas-installer/cmd/root.go

```diff
@@ -23,8 +37,7 @@ var rootCmd = &cobra.Command{
 	For a successful installation, you must have provide the path to a kubeconfig file, or have 
 	one in $HOME/.kube/config. Additionally, the following CLI tools must all be in your $PATH environment.
 	
-	oc - used to install Keycloak
-	openssl - used to create the root certificates	`,
+	oc - used to install Keycloak `,
```

Comment:
Doesn't this PR remove Keycloak and not openssl?

Reply:
yea, this was leftover from the certs PR that merged today